bluej/src/main/java/bluej/editor/flow/FlowEditorPane.java

```diff
@@ -703,7 +715,7 @@ public void positionCaret(int position, EditType editType)
     {
         if (editType == EditType.NON_IME_EDIT)
         {
-            nonIMEdit();
+            //nonIMEdit();
```

Comment:
Does this commented code need to be removed since it's now invalidated?

Reply:
Yes, good point.  I've removed it, which also means I could remove the editType paramter to that method.